src/main/java/com/redhat/jenkins/plugins/bayesian/User.java

```diff
@@ -0,0 +1,49 @@
+package com.redhat.jenkins.plugins.bayesian;
+
+
+import java.util.List;
+
+
+public class User {
```

Comment:
This class is full of tabs :) Please switch to spaces to stay consistent with the rest of the codebase.